lepton/starterkits/FLUX_NIMs_Interactive_Lab.ipynb

```diff
@@ -0,0 +1,2674 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# FLUX NIMs Interactive Lab\n",
+    "\n",
+    "Welcome to hands-on exploration of NVIDIA NIMs with FLUX image generation models!\n",
+    "\n",
+    "## What are NVIDIA NIMs?\n",
+    "NVIDIA NIMs (NVIDIA Inference Microservices) are optimized containers that serve AI models with maximum performance. Instead of downloading large model files locally, you connect to pre-optimized endpoints that handle inference with advanced optimizations like quantization (FP8/FP4) and TensorRT acceleration.\n",
+    "\n",
+    "## What You'll Do:\n",
+    "- 🚀 Deploy FLUX.1-dev NIM to the cloud using Lepton\n",
+    "- 🎨 Generate stunning images from text prompts\n",
+    "- 🔧 Experiment with different parameters (steps, seed, prompts)\n",
+    "- ✂️ Edit images with natural language using FLUX Kontext\n",
+    "- 💡 Learn production-ready AI deployment workflows"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 1: Environment Setup\n",
```

Comment:
Just putting a note up here for now, but for consistency, can you update all of the non-title headers to be level-3 or below? So this would be `### Step 1...`. Any sub-sections after these headers should be decreased by one as well.